src/styles/main.scss

```diff
@@ -1,3 +1,216 @@
+html {
+  font-weight: 500;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+/* #region header */
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+}
+
+.logo__img {
+  display: block;
+  width: 100%;
+  cursor: pointer;
+}
+
+.nav__list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: inline-block;
+  margin-right: 17px;
+}
+
+.nav__item:last-of-type {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  box-sizing: border-box;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #060b35;
+  font: inherit;
+  position: relative;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+/* #endregion header */
+
+.main {
+  width: 944px;
+  margin: 0 auto;
+  padding: 50px 0;
+}
+
+.content {
+  display: flex;
+  flex-wrap: wrap;
+  font-weight: 400;
+  color: #060b35;
+}
+
+/* #region product */
+
+.product {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  margin-right: 48px;
+  margin-bottom: 46px;
+}
+
+.product:nth-child(4n) {
+  margin-right: 0;
+}
+
+.product:nth-last-child(-n + 4) {
+  margin-bottom: 0;
+}
```

Comment:
when I add a new card it should work correctly 
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/53611910/197878304-ecfbcb25-f57c-413b-9e05-d068e7773475.png">